layouts/partials/about.html

```diff
@@ -1,20 +1,20 @@
 <h2 class="section-title">{{ .titles.about }}</h2>
 
 <p itemprop="description">
-  Pour cette toute première édition, la conférence en science des données appliquée au hockey de
-  Montréal (SDHMTL) a comme objectif de promouvoir l'utilisation de la science des données dans
-  l'univers du hockey.
+  La conférence en science des données appliquée au hockey de Montréal (SDHMTL) est un événement unique en
+  son genre au Québec qui a pour but de réunir un large spectre de passionné(e)s de hockey.
```

Comment:
Je pense que j'aime mieux utiliser le · pour le language inclusif, e.g. passionné·e·s, c'est plus tendance. What do you think?

Reply:
Ouais bien d'accord.